lib/sul_styles/engine.rb

```diff
@@ -0,0 +1,11 @@
+module SulStyles
+  class Engine < ::Rails::Engine
+    isolate_namespace SulStyles
+
+    config.after_initialize do |app|
+      unless Rails.env.production?
```

Comment:
Ok. I don't like mounting it at `/`, though. Why not mount it at `/styleguides` (or whatever) and have the engine set the root route?